--- elasticsearch.orig	2017-08-12 11:17:17.000000000 +0000
+++ elasticsearch	2017-08-29 04:44:48.765845000 +0000
@@ -105,7 +106,7 @@
         debug "pid file ($_pidfile): no pid in file."
         return
     fi
-    if [ -n "`/usr/local/bin/jps -l | grep -e "^$_pid"`" ]; then
+    if [ -n "`/usr/local/bin/jps -l | grep -v 'process information unavailable' | grep -e "^$_pid"`" ]; then
         echo -n $_pid
     fi
 }
